toggl/api/base.py

```diff
@@ -222,7 +222,10 @@ def _fetch_all(self, url, order, config):  # type: (str, str, utils.Config) -> t
         Helper method that fetches all objects from given URL and deserialize them.
         """
         fetched_entities = utils.toggl(url, 'get', config=config)
-
+        
+        if isinstance(fetched_entities, dict):
+            fetched_entities = fetched_entities.get('data')
```

Comment:
Hmm I am wondering if it wouldn't make more sense to have this in `utils.toggl`? I guess the main fetching (eq. `GET` requests) are coming from the `TogglSet` but maybe it would be applicable also for other responses? WDYT?

Reply:
I really have no strong opinion here since I only made these changes to get a broken tool using this project up and running again and have very little context. 

But if I read this correct, https://github.com/AuHau/toggl-cli/blob/feat/new-api-version/toggl/utils/others.py#L149 , it should return an dict right?
And I think the problem was that for calls without the envelop, it return a list.

I do think it would be easier to discuss that on the other PR once all changes needed for V9 are identified.

Reply:
Sounds good!